.github/workflows/container-build-and-push.yaml

```
@@ -0,0 +1,24 @@
+name: Build and push image
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - master
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    steps:
+      - id: build-push-image
+        uses: castoredc/container-build-push-action@v1
+        with:
+          image: "castor/senchacmd"
```

I think this should be 
```suggestion
          image: "cdms/senchacmd"
```
or alternatively `dev-tools/senchacmd`. 

We'll also need to update https://github.com/castoredc/castor-infrastructure/blob/8262fe89e94e0f66fc3d7c88ae7c9e0241b1b1a1/terraform/environment/management/deployments/registry/registry.tf to add the image under the right `repositories_read_write` entry to ensure the acr-username used below is allowed to write to the image :)

---

Done here! https://github.com/castoredc/castor-infrastructure/pull/4057